src/tutorial/store/Item.java

```diff
@@ -0,0 +1,50 @@
+
+
+public class Item {
+	
+	
+	public Item(String itemName, String description, String storeDetails, int price) {
+		super();
+		ItemName = itemName;
+		Description = description;
+		StoreDetails = storeDetails;
+		this.price = price;
+	}
+
+	private  String ItemName,Description,StoreDetails;
+	private  int price;
```

Comment:
Coding guide lines
Private variable are usually declared on top before constructor.
Starting letter is small for variable names.
It is good practice to add one variable per line